rules/cyberark/privilege_escalation_cyberarkpas_error_audit_event_promotion.toml

```diff
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2021/06/23"
+maturity = "production"
+updated_date = "2021/06/23"
+min_stack_version = '7.14.0'
+
+[rule]
+author = ["Elastic"]
+description = "Identifies the occurrence of a CyberArk Privileged Access Security (PAS) error level audit event."
+false_positives = ["To tune this rule, add exceptions to exclude any event.code which should not trigger this rule."]
```

Comment:
Please explain either here or in the description what `event.code` means. By excluding an `event.code` does that mean that I'm suppressing a rule on that side?

Reply:
add to description b3ddf345ef10c058615be1585b5994e0103ca40b